### DISCUSSION TITLE
How can I access the Pi via SSH while connected to the hotspot via wifi?

### DISCUSSION THREAD

**Discussion by theycallmevirgo** [RaspAP/raspap-webgui#1181] [Q&A] [answered]
I can connect via wifi and pull up the configuration page but I'd also like to connected via ssh. Is this supported via the wifi connection or only via ethernet? If ssh is supported, what address and account  should I use? Can I use my original account or must I use the raspap admin account? Does the raspap admin account have sudo rights?

**Comment by billz** [comment, marked as answer]
While connected to the hotspot via Wi-Fi, you may access your device with SSH on the `10.3.141.1` static IP address. For example, with a "pi" user account this would be `ssh pi@10.3.141.1`. If your device is connected via ethernet, you may also access it over your local network via SSH. 

The RPi OS kernel includes the avahi-daemon which facilitates local network discovery via multicast DNS (mDNS). On client computers with the Bonjour service installed (all macOS machines and Windows PCs with Apple iTunes), you can access your device via SSH with `ssh pi@raspberrypi.local`. If the .local domain address doesn't resolve for you, check your router's list of connected devices.

RaspAP does not create any Linux users; only the www-data user that lighttpd runs under is granted [very specific rights](https://github.com/RaspAP/raspap-webgui/blob/master/installers/raspap.sudoers) to enable administration via the web UI.

**Comment by catfoobar** [comment]
Tried the above (ssh user@10.4.141.1) and get connection refused - what could be wrong?

**Reply by billz** [reply]
Try using the correct `10.3.141.1` address 😉

**Comment by catfoobar** [comment]
: )  Fat fingered the above  -

```
dan@Spot:~$ ssh 10.3.141.1
ssh: connect to host 10.3.141.1 port 22: Connection refused
dan@Spot:$ ssh 10.3.141.1

dan@Spot:~$ ping 10.3.141.1
PING 10.3.141.1 (10.3.141.1) 56(84) bytes of data.
--- 10.3.141.1 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4091ms
```
First ssh above connected over wireless connection.
Second ssh above over wired connection.
All within local network.

Am able to view admin screen web page if connected over wireless.

Going to reset home router and plug pi into display/keyboard to check integrity.

**Reply by catfoobar** [reply]
Oh.  Need to explicitly enable ssh on Pi running AP    : )

Cheers!
